webapp/channels/src/components/announcement_bar/users_limits_announcement_bar/index.tsx

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {useIntl} from 'react-intl';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {ExclamationThickIcon} from '@mattermost/compass-icons/components';
+import type {ClientLicense} from '@mattermost/types/config';
+
+import {getUsersLimits} from 'mattermost-redux/selectors/entities/limits';
+
+import {openModal} from 'actions/views/modals';
+
+import AirGappedContactSalesModal from 'components/air_gapped_contact_sales_modal';
+import AnnouncementBar from 'components/announcement_bar/default_announcement_bar';
+import useCWSAvailabilityCheck, {CSWAvailabilityCheckTypes} from 'components/common/hooks/useCWSAvailabilityCheck';
+
+import {AnnouncementBarTypes, LicenseLinks, ModalIdentifiers} from 'utils/constants';
+
+type Props = {
+    license?: ClientLicense;
+};
+
+function UsersLimitsAnnouncementBar(props: Props) {
+    const dispatch = useDispatch();
+
+    const {formatMessage} = useIntl();
+
+    const cwsAvailability = useCWSAvailabilityCheck();
+    const usersLimits = useSelector(getUsersLimits);
+
+    const handleCTAClick = useCallback(() => {
+        if (cwsAvailability === CSWAvailabilityCheckTypes.Available || cwsAvailability === CSWAvailabilityCheckTypes.Unknown) {
```

Comment:
Seems that with the current implementation of `useCWSAvailabilityCheck` we are currently not checking if it's airgapped for TE builds (!isEnterpriseReady). 

Wondering if we should test against another url for airgapped that is not CWS given that I believe the CWS checks go through Enterprise code (reason why it doesn't check in TE) IIRC. Should we just try to ping something like google.com?

Reply:
i posted this question in community https://community.mattermost.com/core/pl/wgxxfykh1jfkbqa8o3xwn8ntzo

Reply:
Thinking about this more and after @hmhealey comment in that thread, we can still test against portal url (which is in the config) but we should find a way to do this without going through the enterprise endpoints so it can also work for team edition

Reply:
We discussed this in our planning today and we don't consider this case a blocker as the link will open for team edition servers that are airgapped in another browser where they can still copy from.

Reply:
👍